moarchiving/moarchiving.py

```diff
@@ -569,6 +609,33 @@ def hypervolume(self):
             assert abs(self._hypervolume - self.compute_hypervolume(self.reference_point)) < 1e-12
         return self._hypervolume
 
+    @property
+    def hypervolume_plus(self):
```

Comment:
I am not sure my issue is resolved: what _is_ the hypervolume_plus / how can I find out what it is?